samples/baseline_esxi_config.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+import vm_automation
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-k", "--keyword", help="VM search parameter")
+    parser.add_argument("-h", "--hostname", help="New hostname for the VM")
+    parser.add_argument("-sn", "--snapshotName", help="snapshot name")
+    parser.add_argument("hypervisorConfig", help="json hypervisor config")
+    
+    args = parser.parse_args()
+
+    vmServer = vm_automation.esxiServer.createFromFile(args.hypervisorConfig, './power.log')
+    if vmServer != None:
+        vmServer.enumerateVms()
+        for vm in vmServer.vmList:
+            if (args.keyword == None) or (args.keyword in vm.vmName):
+                # Change hostname
+                assert subprocess.call('python manageHostname.py ' + args.hypervisorConfig + ' -k ' + args.keyword + ' -h ' + args.hostname) == 0
+
+                # Snapshot VM
+                assert subprocess.call('python Snapshots.py ' + args.hypervisorConfig + ' -k ' + args.keyword + ' -sn ' + args.snapshotName + ' -a create -p0') == 0
```

Comment:
Calls the wrong script for snapshot.

This file is not a sample for how to utilize the vm-automation library, please remove it as it does not provide a strong example of why using this library provides value.

Can be performed just as easily directly on the command line or the end user can create their own wrapper.
```
python manageHostname.py  esxi_config_file.json -k name_filter -h new_hostname
python manageSnapshots.py  esxi_config_file.json -k name_filter -sn snaphost_name
```